examples/step-22/step-22.cc

```diff
@@ -976,10 +982,12 @@ namespace Step22
       std::vector<unsigned int> subdivisions(dim, 1);
       subdivisions[0] = 4;
 
-      const Point<dim> bottom_left =
-        (dim == 2 ? Point<dim>(-2, -1) : Point<dim>(-2, 0, -1));
-      const Point<dim> top_right =
-        (dim == 2 ? Point<dim>(2, 0) : Point<dim>(2, 1, 0));
+      const Point<dim> bottom_left = (dim == 2 ?             //
```

Comment:
same here, but others might have different opinions.

Reply:
You could say "2d case" and "3d case" behind the `//` if you wanted.